pkg/epp/flowcontrol/registry/config.go

```diff
@@ -112,9 +112,9 @@ type Config struct {
 // that operate at this priority level.
 type PriorityBandConfig struct {
 	// Priority is the unique numerical priority level for this band.
-	// Convention: Lower numerical values indicate higher priority (e.g., 0 is highest).
+	// Convention: Lower numerical values indicate lower priority (e.g., 0 is lowest).
```

Comment:
0 is not the lowest, 0 is the midpoint.

Reply:
Right, thanks for the catch. Updated the PR